src/newgrf_spritegroup.cpp

```diff
@@ -154,6 +154,7 @@ static uint32 RotateRight(uint32 val, uint32 rot)
 {
 	/* Do not rotate more than necessary */
 	rot %= 32;
+	assert(rot > 0);
```

Comment:
Is "== 0" valid?

Reply:
The point it that it shouldn't be, as then you're doing `val << 32`, which isn't defined all that well. If we want to just assume that it should be 0, that's a bit different.

That said, not sure why this exists at all, when ROR in bitmath_func.hpp exists.